digdag-docs/src/operators/s3_wait.md

```diff
@@ -81,6 +81,21 @@ For more information about SSE-C, See the [AWS S3 Documentation](http://docs.aws
   An optional flag to control whether to use path-style or virtual hosted-style access when accessing S3.
   *Note:* Enabling `path_style_access` also requires specifying a `region`.
 
+* **timeout**: TIMEOUT
+
+  Set timeout.
+
+  Examples: wait 120 seconds
+
+  ```
+  timeout: 120s
+  ```
+
+* **ignore_timeout_error**: true/false (default:false)
+
```

Comment:
PollingWaiter is used also by EmrOperatorFactory, BqJobRunner, and GcsWaitOperatorFactory. At least GcsWaitOperatorFactory should have the same option. Consistency across different but similar operators is important for usability.
What is your strategy to expand the same option to those other operators?

Reply:
And AbstractWaitOperatorFactory (which is a base class of td_wait and td_wait_table).

Reply:
My thought is the impact of modification to be minimized to avoid unexpected degradation.
I added the parameters, so other operators can change easily later in my understand.
But I understand your opinion.

> Consistency across different but similar operators is important for usability.

Reply:
ok. Do you mean that the same option should be added to other similar operators but not now? If so, it sounds good. Which operators do you expect to have the option?

Reply:
Yes. I guess gcs_wait> should also hove same option. 
td_wait> and td_wait_table> may be also good with timeout option.

Reply:
Sounds good. For gcs_wait>, it's good idea to add a comment somewhere. It would say something like...`// timeout 
 and continue_on_timeout should be implemented later here. See https://github.com/treasure-data/digdag/pull/1280/files#r358679909 for details`. (Otherwise we will forget)

Reply:
I filed a ticket in #1281 and our internal ticket system for gcs_wait> timeout.